src/app/pages/ArticlePage/index.stories.tsx

```diff
@@ -229,6 +230,20 @@ export const ArticlePageWithPodcastNews = () => (
   />
 );
 
+export const ArticlePageWithTranscriptStage2 = () => (
+  <ComponentWithServiceContext
+    data={articleDataWithTranscript}
+    service="mundo"
+  />
+);
+
+export const ArticlePageWithTranscriptStage3 = () => (
```

Comment:
As above, how do we indicate what stage 3 means without having to look elsewhere in the code - in this case, is it no transcript?